frontend/src/contexts/wishlist-context.tsx

```diff
@@ -44,33 +54,96 @@ const WishlistContextProvider: React.FC<{ children: ReactNode }> = ({
     const isLiked = likedCars?.includes(carId as string);
 
     isLiked ? handleDeleteWishlist(data) : handleCreateWishlist(data);
+
+    if (isLiked) {
+      const existing = notifications.find(
+        (n) =>
+          n.modelId === data.modelId ??
+          n.complectationId === data.complectationId,
+      );
```

Comment:
Why do you use `??` here? `n.modelId === data.modelId` always returns **a defined** boolean

Reply:
it seems to me that we sometimes like a model of the car and sometimes an exact complectation of the car, so thats why we have such interface for liked cars 
![image](https://user-images.githubusercontent.com/82455714/189497865-50da4911-3515-4709-9bae-099c1b3a8b9e.png)

Reply:
No, I mean that exactly the `??` is redundant here. Because ` n.modelId === data.modelId ?? n.complectationId === data.complectationId` is getting compiled to the `boolean ?? boolean`. But `??` is used as the `&&` operator but it doesn't return false if on the left side you'll have a falsy value (`''`, `0`, `NaN`). In our case `&&` would be more appropriate.

Also, if you target your TS compiler to ES5, the `??` will generate much more code because of the transpilation: 
![image](https://user-images.githubusercontent.com/68850090/189519081-c3488b97-08b6-4777-bb43-e944483a9f98.png)